src/src/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -44,7 +44,8 @@
      */
     private String mSharedPrefPackageName;
 
-    private boolean mSkipBroker = false;
+    @Deprecated private boolean mSkipBroker = false;
```

Comment:
Have you tried to use refactoring/rename variable?
deprecate private variable seems useless...just remove it